Snake/src/main/java/org/example/state/State.java

```diff
@@ -35,12 +57,14 @@ private void processInput() {
 
     private void update() {
         snake.update();
+        apple.update();
     }
 
     private void draw() {
         BeginDrawing();
         ClearBackground(BLACK);
         snake.draw();
+        apple.draw();
```

Comment:
tutaj lepiej dać apple.draw() nad snake.draw(), to jest kolejność rysowania, więc aktualnie jabłko będzie wyżej niż wąż co możę trochę gorzej wyglądać jak wąż będzie przez nie przechodzić